index.html

```diff
@@ -191,27 +193,21 @@ <h2 class="section-header">About Myself</h2>
       </section>
 
       <section class="contact" id="contact">
-        <div class="contact-wrapper">
-          <h3 class="section-header">Contact Me</h3>
-          <h5>
-            If you have an application you are interested in developing,
-            a feature that you need built or a project that needs coding.
-            I’d love to help with it.
-          </h5>
-          <form action="#" class="contact-form">
-            <input type="text" name="name" id="name" placeholder="Name">
-            <input type="email" name="email" id="email" placeholder="E-mail">
-            <textarea 
-            name="message" 
-            id="message" 
-            cols="30" 
-            rows="5" 
-            placeholder="Write your message here"></textarea>
-            <button type="submit" class="buttons">Get In Touch</button>
+        <div class="contact_wrapper">
+          <h2 class="section-header">Contact me</h2>
+          <p class="description">If you have an application you are interested in developing,
+            a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          <form action="#" method="post">
```

Comment:
- [ ] Please make sure the form uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html).
* `action` value should be similar to `https://formspree.io/f/{form_id}`

**This will allow us to use formspree service to collect the form data and our users will be able to contact us using the form in our portfolio page.**